src/content/components/FeatureView/FeatureView.js

```diff
@@ -248,10 +286,13 @@ export class FeatureView extends React.PureComponent {
   sortByRelease(bugs, subMetas) {
     const result = {
       untriaged: [],
+      untriagedBySubMeta: {},
```

Comment:
I think actually it's ok if `untriaged` doesn't get split up by sub meta, there's usually not enough bugs to warrant spliting them up